Docs: remove callouts from docs (#1820)
vtex/faststore: apps/site/pages/docs/getting-started/5-next-steps.mdx

<Callout type="warning" emoji="⚠️">
This documentation is currently under development.
</Callout>

Now that you have your FastStore project up and running, it's important to take advantage of additional resources to enhance your store experience:

> ## [Theming](/docs/brandless)

Author: ```suggestion
## Theming
```

> Modify the appearance and user interface of your store with Theming. By customizing the theme, you can give your store a unique look and feel that aligns with your brand.

Author: ```suggestion
Modify your store's appearance and UI with [Theming](/docs/brandless). Customizing the theme gives your store a unique look and feel that aligns with your brand.
```

## VTEX Headless CMS (coming soon!)

Manage your store content with our preferred solution for content management. This will give editors the autonomy to create, edit, and publish web content via the VTEX Admin.

> ## [Go-live](/docs/go-live/overview)

Author: ```suggestion
## Go-live
```

> Integrate your FastStore project with VTEX and get ready to launch your store. This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless checkout experience for your customers.

Author: ```suggestion
Integrate your FastStore project with VTEX and get ready to [launch your store](/docs/go-live/overview). This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless customer checkout experience.
```

## Component customization

Explore the various ways you can extend and customize the components of your store. This could involve modifying existing components or creating new ones to meet your specific requirements. By customizing components, you can tailor your store's functionality to suit your business needs.

> ## [UI components](/components)

Author: ```suggestion
## UI components

```

> Take advantage of the FastStore library components, which are designed to help you quickly implement and customize features in your store.

Author: ```suggestion
Take advantage of the [FastStore UI library components,](/components)
 designed to help you quickly implement and customize features in your store.
```
